src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -570,8 +574,7 @@ void ProblemManager::generateMesh()
         if( order > 1 )
         {
           MeshLevel & mesh = meshBody.createMeshLevel( MeshBody::groupStructKeys::baseDiscretizationString(),
-                                                       discretizationName,
-                                                       order );
+                                                       discretizationName, cellBlockManager, order );
```

Comment:
Thanks for the comment. Yes, the if statement here "if( order > 1 )" is to "create a high-order MeshLevel" by calling "meshBody.**createMeshLevel**". I think it would not discriminate "order=1" though, because as in line 585, "else if( order==1 )" would call for a different function "meshBody.**createShallowMeshLevel**". Please feel free to let us know if I misunderstand the question. We can address it further in that case.